### PR TITLE
Update expo sdk to 28.0.0, snack-sdk to 2.0.0

### PR DESCRIPTION
**Pull request by cpirich**
* Expo no longer supports v25 of their SDK. Upgraded to `28.0.0` (`32.0.0` is the latest, but our `CustomAsset` solution is not compatible with anything after `28.0.0`)
* Upgrade `snack-sdk` to `2.0.0` (no longer use our private `@code-dot-org/snack-sdk` package, which was only intended to be temporary)
* Remove an export template file that is no longer needed
* Fix a few `fontFace` styles to be `fontFamily`
* Adjust exported app's bundle ids to not include the word `applab` (since we now use this for both applab and gamelab)
* Tested with "Try In Expo" and by exporting as a ZIP and building with expo command line tools (running in both iOS and Android emulators with the latest version of Expo)

### CODE REVIEW

File: apps/package.json [hunk 1]
```diff
@@ -215,7 +215,7 @@
   },
   "dependencies": {
     "@code-dot-org/js-interpreter": "^1.3.6",
-    "@code-dot-org/snack-sdk": "^1.0.11",
+    "snack-sdk": "2.0.0",
```
Comment (islemaster) [inline apps/package.json:218]: :tada: Hooray convergence!

File: apps/src/applab/Exporter.js [hunk 2]
```diff
@@ -569,7 +569,7 @@ export default {
       sessionId: `${getEnvironmentPrefix()}-${project.getCurrentId()}`,
       files,
       name: project.getCurrentName(),
-      sdkVersion: '25.0.0',
+      sdkVersion: '28.0.0',
```
Comment (islemaster) [inline apps/src/applab/Exporter.js:572]: Do you have a sense of how soon we'll have to address the issue preventing us from upgrading past v28?

v32 - January 2019
v31 - November 2018
v29 - July 2018

Having trouble finding a complete history, but it looks like they're releasing a major version every two months?  So we should assume we have six months to upgrade again?

File: apps/src/templates/export/expo/app.json.ejs [hunk 3]
```diff
@@ -14,6 +14,9 @@
       "resizeMode": "contain",
       "backgroundColor": "#ffffff"
     },
+    "updates": {
+      "fallbackToCacheTimeout": 0
+    },
```
Comment (islemaster) [inline apps/src/templates/export/expo/app.json.ejs:19]: What does this do?

**Review by islemaster — APPROVED** [review]
Nice!

File: apps/src/templates/export/expo/app.json.ejs (hunk 3 above)
Comment (cpirich) [reply apps/src/templates/export/expo/app.json.ejs:19]: I generated a new project with a newer SDK and it included this by default. It is their recommended value to ensure that app updates occur asynchronously (impacted the next launch of the app)

File: apps/src/applab/Exporter.js (hunk 2 above)
Comment (cpirich) [reply apps/src/applab/Exporter.js:572]: Yes, that sounds about right. I hope to discuss with them soon to confirm.

**Pull request merged**